lib/win32/backend/policies.rb

```diff
@@ -0,0 +1,44 @@
+#

```

Comment:
This file contains methods for validation. @btm, @mwrock , please share your views if the file name `policies` sounds good to you.